api/src/home/analysis-platform/job_handlers/azure_container_instances/__init__.py

```diff
@@ -68,20 +68,25 @@ def start(self) -> str:
         for env_string in self.job.entity.get("environmentVariables", []):
             key, value = env_string.split("=", 1)
             env_vars.append(EnvironmentVariable(name=key, value=value))
-        runnerEntity: dict = self.job.entity["runner"]
+        reference_target: str = self.job.entity["referenceTarget"]
```

Comment:
IMO, this field should be optional, so both this job_handler and srs_wrapper.py should handle that